codec-http2/src/main/java/io/netty/handler/codec/http2/StreamBufferingEncoder.java

```diff
@@ -52,6 +54,15 @@
 public class StreamBufferingEncoder extends DecoratingHttp2ConnectionEncoder {
 
     /**
+     * Thrown if buffered streams are terminated due to this encoder being closed.
+     */
+    public static final class ChannelClosedException extends Http2Exception {
```

Comment:
maybe a nit but should we change the class name to not confuse it with the one existing in java.nio itself ? Maybe Http2ChannelClosedException